src/app/loadout-builder/hooks/useProcess.ts

```diff
@@ -199,34 +209,61 @@ function mapDimSocketToProcessSocket(dimSocket: DimSocket): ProcessSocket {
   };
 }
 
-function mapSeasonalModsToSeasonsArray(
-  lockedSeasonalMods: readonly LockedModBase[]
-): ProcessModMetadata[] {
+function mapSeasonalModsToSeasonsArray(lockedSeasonalMods: readonly LockedModBase[]): ProcessMod[] {
```

Comment:
As always I like seeing documentation for these

Reply:
Will do